commands/account/auth.ts

```diff
@@ -0,0 +1,183 @@
+    }
+
+    if (doesConfigExist) {
+      updateAccountConfig({
+        ...updatedConfig,
+        environment: updatedConfig.env,
+        tokenInfo: updatedConfig.auth!.tokenInfo,
+      });
+      writeConfig();
+    }
+
+    return updatedConfig as CLIAccount_NEW;
+  } catch (e) {
+    logError(e);
+    return null;
+  }
+}
+
+export const describe = null; // i18n(`${i18nKey}.describe`);
+export const command = 'auth';
+
+type AccountInfoArgs = CommonArgs & ConfigArgs;
+
+export async function handler(
+  args: ArgumentsCamelCase<AccountInfoArgs>
+): Promise<void> {
+  const { providedAccountId, disableTracking } = args;
+
+  if (!disableTracking) {
+    trackCommandUsage('account-auth', {}, providedAccountId);
+  }
+
+  const env = args.qa ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
+  const configExists = configFileExists(true);
+
+  if (!configExists) {
+    createEmptyConfigFile({}, true);
+  }
+  loadConfig('');
+
+  handleExit(deleteEmptyConfigFile);
+
+  try {
+    const updatedConfig = await createOrUpdateConfig(
+      env,
+      configExists,
+      providedAccountId
+    );
+
+    if (!updatedConfig) {
+      logger.error(i18n(`${i18nKey}.errors.failedToUpdateConfig`));
+      process.exit(EXIT_CODES.ERROR);
+    }
+
+    const { name, accountId } = updatedConfig;
+
+    // If the config file was just created, we don't need to prompt the user to set as default
+    if (!configExists) {
+      logger.log('');
+      logger.success(
+        i18n(`${i18nKey}.success.configFileCreated`, {
+          configPath: getConfigPath('', true)!,
+        })
+      );
+      logger.success(
+        i18n(`${i18nKey}.success.configFileUpdated`, {
+          account: name || accountId,
+        })
+      );
+    }
+
+    if (configExists) {
+      const setAsDefault = await setAsDefaultAccountPrompt(name!);
+
+      logger.log('');
+      if (setAsDefault) {
+        logger.success(
+          i18n(`lib.prompts.setAsDefaultAccountPrompt.setAsDefaultAccount`, {
+            accountName: name!,
+          })
+        );
+      } else {
+        logger.info(
+          i18n(`lib.prompts.setAsDefaultAccountPrompt.keepingCurrentDefault`, {
+            accountName: getConfigDefaultAccount()!,
+          })
+        );
+      }
+    }
+    uiFeatureHighlight(['helpCommand', 'authCommand', 'accountsListCommand']);
+
+    process.exit(EXIT_CODES.SUCCESS);
+  } catch (err) {
+    logError(err);
+    process.exit(EXIT_CODES.ERROR);
+  }
+}
+
+export function builder(yargs: Argv): Argv<AccountInfoArgs> {
+  yargs.options({
+    account: {
+      describe: i18n(`${i18nKey}.options.account.describe`),
+      type: 'string',
+      alias: 'a',
+    },
+    'disable-tracking': {
+      type: 'boolean',
+      hidden: true,
+      default: false,
+    },
+  });
+
+  addConfigOptions(yargs);
+  addTestingOptions(yargs);
+  addGlobalOptions(yargs);
+
+  return yargs as Argv<AccountInfoArgs>;
```

Comment:
Looks like a small copy/paste miss re: `AccountInfoArgs`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1365/commits/5529392be0664cf948ca893bd6c95af2d373d9c1